test/selenium/lib/view/signed_in/view.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+var SignedInPageAccessor = require('./accessors.js');
+var SignedOutPageView = require('../signed_out/view.js');
+var signedOutPageView;
+
+function SignedInPageView(driver) {
+    this.driver = driver;
+    this.accessors = new SignedInPageAccessor(this.driver);
+    signedOutPageView = new SignedOutPageView(this.driver);
```

Comment:
In this case maybe not. I would save one line of code if more functions had to return a signedOutPageView, but is not the case in this test...